src/App.tsx

```diff
@@ -8,55 +8,131 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
+import { User } from './types/User';
+import { Post } from './types/Post';
+import { getUserPosts, getUsers } from './api/client';
+import { formatPosts, formatUsers } from './utils/formatData';
+import { UserResponse } from './types/UserResponse';
+import { Notification } from './components/Notification';
+import { PostResponse } from './types/PostResponse';
+import { NewCommentForm } from './components/NewCommentForm';
+import { Comment } from './types/Comment';
 
 export const App: React.FC = () => {
+  const [users, setUsers] = useState<User[]>([]);
+  const [user, setUser] = useState<User>();
```

Comment:
```suggestion
  const [user, setUser] = useState<User | null>(null);
```

use null as a default value